src/AgGrid.elm

```diff
@@ -111,6 +111,7 @@ type Renderer dataType
     | NoRenderer
     | SelectionRenderer (dataType -> String) (List String)
     | StringRenderer (dataType -> String)
+    | GroupRenderer (dataType -> String)
```

Comment:
We should add some documentation about this "master-detail" feature. So people will know that we support this feature and how they can use it.